packages/UI/package.json

```diff
@@ -20,6 +20,7 @@
     "react-burger-menu": "^3.0.8",
     "react-dom": "^17.0.2",
     "react-scripts": "5.0.1",
+    "storybook-addon-designs": "6.3.1",
```

Comment:
This plugin allows you to render a component coming from figma, linked to a specific story.

for example you can link `DefaultChecked` story to the same exact state from Figma and then designers would be able to compare the design to the implementation right within storybook.
here's the [docs](https://storybook.js.org/addons/@storybook/addon-designs/)